pm/recipient_test.go

```diff
@@ -102,30 +102,41 @@ func TestReceiveTicket_InvalidRecipientRand(t *testing.T) {
 	require.NotEqual(t, params.FaceValue, ticket.FaceValue)
 	assert.Equal(sessionID, "")
 	assert.Equal(won, false)
-	assert.Equal(err, errInvalidTicketRecipientRand)
+	assert.Equal(err.Error(), errInvalidTicketRecipientRand.Error())
+	_, ok := err.(*FatalReceiveErr)
+	assert.True(ok)
```

Comment:
Let's add test cases for errors that are not instances of `FatalReceiveErr` i.e. `errInvalidTicketSignature`, expired `ticket.ParamsExpirationBlock`, error when an invalid recipientRand is revealed and error when senderNonce is already used.

Reply:
fixed in 5b26487